scripts/netContain/scripts/contivNet.sh

```diff
@@ -135,7 +140,7 @@ if [ $netmaster == true ]; then
 	done
 elif [ $netplugin == true ]; then
 	echo "Starting netplugin"
-	modprobe openvswitch
+	modprobe openvswitch || die "failed to load openvswitch kernel module"
```

Comment:
Would including `$?` in the output be helpful?  I'm not sure how many exit codes `modprobe` can return lol